packages/create-astro/src/index.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
```

Comment:
What's this file about?

Reply:
Oops, probably missed this in the rebase.